cli/app/app.go

```diff
@@ -234,18 +234,22 @@ func ProjectDelete(p project.APIProject, c *cli.Context) error {
 		RemoveVolume: c.Bool("v"),
 	}
 	if !c.Bool("force") {
-		options.BeforeDeleteCallback = func(stoppedContainers []string) bool {
```

Comment:
Hum, I would have kept the `BeforeDeleteCallback` (which might have the wrong name now I think of it…) that could allow interactive stuff from the lib… but that's not really a big deal and that might complicate thing where there is no need to, so ok :angel: